webos-dart/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "sample-webos-dart",
+    "description": "Sample localization project",
+    "repository": "git@github.com:iLib-js/ilib-loctool-samples.git",
+    "license": "Apache-2.0",
+    "version": "1.0.0",
+    "scripts": {
+        "loc": "loctool -2 --xliffStyle custom --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB --pseudo --localizeOnly",
+        "debug": "node --inspect-brk node_modules/loctool/loctool.js -2 --xliffStyle custom --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB --pseudo --localizeOnly",
+        "clean": "rm -rf *.xliff assets",
+        "test": "node test/testResources.js"
+    },
+    "dependencies": {
+        "ilib-loctool-webos-json-resource": "^1.5.8",
```

Comment:
Need to npm add ilib-loctool-webos-dart

Reply:
I updated it. but it hasn't been published yet. I'm thinking of publishing v1.0.0 when my current implementation work is done.